Marlin/src/lcd/menu/menu_bed_leveling.cpp

```diff
@@ -159,7 +164,11 @@
   //         Move to the first probe position
   //
   void _lcd_level_bed_homing_done() {
-    if (ui.should_draw()) MenuItem_static::draw(1, GET_TEXT(MSG_LEVEL_BED_WAITING));
+    if (ui.should_draw()) {
+      MenuItem_static::draw(1, GET_TEXT(MSG_LEVEL_BED_WAITING));
+      // Color UI needs a control to detect a touch
+      TERN_(HAS_GRAPHICAL_TFT, touch.add_control(CLICK, 0, 0, TFT_WIDTH, TFT_HEIGHT));
```

Comment:
Note that `should_draw` will be `true` when the touch is received, so briefly there will be a second touch control added to the display. I see that touch controls are removed when the LCD is cleared, but not otherwise, so be sure to always use standard methods to erase the screen.